code/game/objects/machinery/spaceheater.dm

```diff
@@ -122,3 +122,12 @@
 
 
 	cell.use(50*GLOB.CELLRATE)
+
+/obj/machinery/space_heater/radiator
+	name = "radiator"
+	desc = "It's a radiator. It heats the room through convection with hot water. This one has a red handle."
+	icon_state = "radiator"
+	density = FALSE
```

Comment:
I think having some versions of objects being dense and others not being dense for no particular reason is kinda stinky and unclear for the players (like some platforms) but up to you

Reply:
The radiator sprite really shouldn't be dense in this case, it occupies a very small amount of space and will lead to rage if it's dense.